contracts/zero-ex/contracts/src/features/BundleSwapFeature.sol

```diff
@@ -80,11 +80,15 @@ contract BundleSwapFeature is
                 }
             } else {
                 // cannot ErrorBehaviour.STOP or CONTINUE - always has to revert to not lose sent ETH
+                // REVIEW: I think the issue here might occur even for erc20 cases, the swap can fail
```

Comment:
I will add a check which loads balance before and after trade and then compares it

Reply:
Ah, I forgot that I already implemented that check in line 152:

`revert("BundleSwapFeature::_callFeature/INPUT_TOKEN_LEAK");`